src/logger.ts

```diff
@@ -457,9 +458,8 @@ export class Logger {
       fs.accessSync(logFile, fs.constants.W_OK);
     } catch (err1) {
       try {
-        mkdirp.sync(path.dirname(logFile), {
-          mode: '700',
-        });
+        fs.mkdirSync(path.dirname(logFile));
+        fs.chmodSync(path.dirname(logFile), 0o700);
```

Comment:
@maggiben does chmodSync have the same limitation for changing file mode on Windows?

Reply:
I can use `mod` in `mkdirSync` and do it only if NOT win32